soccer/gameplay/GameplayModule.hpp

```diff
@@ -153,6 +153,9 @@ namespace Gameplay
 			Geometry2d::CompositeShape _ourGoalArea;
```

Comment:
It's odd that some of our obstacles are direct ivars while some are shared_ptrs - let's make them all the same.